contracts/xasset-orchestrator/src/lib.rs

```diff
@@ -1,14 +1,15 @@
 #![no_std]
+
 use loam_sdk::derive_contract;
+use loam_sdk::soroban_sdk::{Address, String, Symbol};
```

Comment:
We have been having putting the types fully qualified on the trait. This way you don't need to import here.

The reason for this is because the trait's signatures are copied as is into this file (this way you could use it from another crate) into a huge `impl` block that is the final external API.  This means the types must be in scope so it's much easier to have them fully qualified `loam_sdk::soroban_sdk::Address` on the trait.